site/src/component/SideInfo/SideInfo.scss

```
@@ -31,6 +31,16 @@
   }
 }
 
+.side-info-name {
+  // Force adaptive width to work while keeping the name width the same
```

it would be helpful if the comment was adjusted more to indicate where this issue occurs. I'm not sure if the "while keeping the name width the same" is necessary, but you can keep that if you want
```suggestion
  // Force adaptive width on the professor page to work
```